### PR TITLE
Fixed PEP8 errors and test code.

### PR DESCRIPTION
**Pull request by Skeen**

### CODE REVIEW

File: addrreg/models/__init__.py [hunk 1]
```diff
@@ -132,7 +132,9 @@ class Meta(object):
         default='https://data.gl/najugaq/municipality',
     )
 
-    code = models.PositiveSmallIntegerField(_('Code'), db_index=True)
+    code = models.PositiveSmallIntegerField(
+        _('Code'), db_index=True, unique=True
+    )
```
Comment (danchr) [inline addrreg/models/__init__.py]: What's the motivation for this change?

File: addrreg/migrations/0006_auto_20170920_0842.py [hunk 2]
```diff
@@ -0,0 +1,301 @@
+# -*- coding: utf-8 -*-
```
Comment (danchr) [inline addrreg/migrations/0006_auto_20170920_0842.py]: Please give this migration a more sensible name.

**Review by danchr — COMMENTED** [review]
Please try to keep trivial, no-op changes such as code style and tests separate from changing migrations. In this case, you wouldn't need the migration if you didn't actually change functionality…

File: addrreg/models/__init__.py (hunk 1 above)
Comment (Skeen) [reply addrreg/models/__init__.py]: The test-case `def test_create_duplicate_municipality_fails(self):` seemed to expect uniqueness of the code field.

Comment (danchr) [reply addrreg/models/__init__.py]: Yeah, it was `XFAIL` since we dropped that requirement…

Comment (danchr) [reply addrreg/models/__init__.py]: It's better to just remove the test, and this change along with it.

Comment (Skeen) [reply addrreg/models/__init__.py]: Alright.

File: addrreg/migrations/0006_auto_20170920_0842.py (hunk 2 above)
Comment (Skeen) [reply addrreg/migrations/0006_auto_20170920_0842.py]: Renamed.

**Comment by Skeen** [comment]
The new migrations file: `addrreg/migrations/0006_sumiiffik_to_sumiffiik.py` seems to have been missing, and thus was added. The code did not run, until makemigrations was run.

**Comment by danchr** [comment]
Oh, oops! 😬 I've added a commit with the migration I created — which is different since it does a proper rename rather add remove+add.

**Comment by danchr** [comment]
(Also, Github apparently dislike history editing. I most certainly do not!)

**Comment by danchr** [comment]
Manually rebased onto development.
